src/api_server/src/lib.rs

```diff
@@ -217,6 +240,15 @@ impl ApiServer {
         response
     }
 
+    fn check_for_fatal_error(&mut self, response: &std::result::Result<VmmData, VmmActionError>) {
+        if let Err(e) = response {
+            // Errors considered as fatal are added here
+            if let VmmActionError::LoadSnapshot(_) = e {
```

Comment:
nit: can be shortened as:
```suggestion
        // Errors considered as fatal are added here
        if let Err(VmmActionError::LoadSnapshot(_)) = response {
```

Reply:
Will do